accounts/views.py

```diff
@@ -202,7 +202,7 @@ def complete_profile(request):
 
 
 def logout_view(request):
-    next_site = request.GET.get('next', 'home:index')
+    next_site = request.GET.get('next', 'home:new_index')
```

Comment:
Please don't make any changes in these files. You only have to move the previous page to `/old/` and place the new page in place of that.

Reply:
Ohk